### PR TITLE
remove not found #411

### PR DESCRIPTION
**Pull request by tedteng**
**What this PR does / why we need it**:
remove not found tests from #411

**Which issue(s) this PR fixes**:
Fixes 

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

File: integration-tests/e2e/kubetest/description/1.20/skip.json [hunk 1]
```diff
@@ -1,101 +1,12 @@
-[{ "testcase": "[Flaky]"},
-  { "testcase": "Alpha"},
-  { "testcase": "[Experimental]"},
```
Comment (schrodit) [inline integration-tests/e2e/kubetest/description/1.20/skip.json:3]: I think we should keep this list of tests that are skipped in general.

File: integration-tests/e2e/kubetest/description/1.20/skip.json [hunk 2]
```diff
@@ -1,101 +1,12 @@
...
-  { "testcase": "[Driver: vSphere]"},
-  { "testcase": "[Driver: cinder]"},
-  { "testcase": "[Driver: gluster]"},
```
Comment (schrodit) [inline integration-tests/e2e/kubetest/description/1.20/skip.json:6]: see comment above

**Review by schrodit — CHANGES_REQUESTED** [review]
otherwise lgtm

File: integration-tests/e2e/kubetest/description/1.20/skip.json (hunk 2 above)
Comment (dguendisch) [reply integration-tests/e2e/kubetest/description/1.20/skip.json:6]: Our openstack provisioner is cinder, hence we could remove the cinder skip to have these test executed.
We can safely remove gluster however as we don't run/deploy it.

Comment (tedteng) [reply integration-tests/e2e/kubetest/description/1.20/skip.json:6]: if I understand correctly remove `cinder` and keep `gluster` in skip is that correct ?

Comment (dguendisch) [reply integration-tests/e2e/kubetest/description/1.20/skip.json:6]: yes (removing cinder from skip will result in executing cinder tests -> we want that as we do have cinder csi in place; keeping gluster in skip will result in skipping gluster tests -> we want that as we have no interest in gluster)

**Review by dguendisch — APPROVED** [review]
/lgtm

**Pull request merged**